plugin.video.intergalacticfm/main.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+# Module: default
+# Author: dreamer
+# Created on: 07.10.2018
+# License: GPL v.3 https://www.gnu.org/copyleft/gpl.html
+
+from json import load
+from sys import argv
+from os.path import isfile
+import requests
+
+from urllib import urlencode
+from urlparse import parse_qsl
+
+
+import xbmcgui
+import xbmcplugin
+import xbmcaddon
+from xbmc import log, LOGNOTICE, LOGERROR
+
+__addonid__ = "plugin.video.intergalacticfm"
+base = xbmc.translatePath('special://home/addons/{}/resources/'.format(__addonid__))
+
+# Get the plugin url in plugin:// notation.
+_url = argv[0]
+# Get the plugin handle as an integer number.
+_handle = int(argv[1])
+
+_addon = xbmcaddon.Addon()
+
+
+fm = 'https://www.intergalactic.fm/'
+tv = 'https://www.intergalactic.tv/'
+pn = 'ifm-system/playingnow.json'
+pl = 'playlist.m3u8'
+
+
+#TODO If JSON on GitHub is younger, load that file. Otherwise, load local file.
```

Comment:
This won't be allowed in our repo.

Reply:
Ok, so the only way to push updates of this kind of meta-data would be to update the plugin? We could also host this json somewhere else and pull from there. Why is it not allowed to do this from github?

Reply:
The reason is that we want to review those changes. The only place allowed for online requests are domains where the content is located. In your case intergalactic.fm and intergalactic.tv.

Reply:
Understood. If we ever implement this it'll be through our domains.